src/main/java/seedu/duke/logic/commands/module/ListModuleCommand.java

```diff
@@ -21,12 +21,16 @@ public ListModuleCommand(String argument) {
     @Override
     public void execute(Ui ui, Storage storage, TaskList taskList, LessonList lessonList, ModuleList moduleList)
             throws DukeException, IOException {
-        assert (argument.equalsIgnoreCase("verbose") | argument.isEmpty())
-                : "list module command currently only accepts empty, \"verbose\"";
-        if (argument.equalsIgnoreCase("verbose")) {
-            ui.printModulesWithDetails(moduleList);
+        if (moduleList.isEmpty()) {
+            ui.printMessage("There are no modules in the list.");
```

Comment:
I think this should be handled by an exception.

Reply:
I'm not sure about this. ListTaskCommand and ListLessonCommand both handle it using an if-check and I don't think the list being empty isn't exceptional behaviour, it's normal for the list to be empty?

Reply:
> the list being empty isn't exceptional behaviour

Good point. The current implementation is fine then.